Docs/Practitioners/PdcPractitioner.md

```diff
@@ -0,0 +1,28 @@
+# PdcPractitioner
+
+Holds the board order and licensing information of the medical professional
```

Comment:
Provides information about a medical professional related to board orders and licensure.